packages/xrpl/src/models/transactions/oracleSet.ts

```diff
@@ -142,14 +143,35 @@ export function validateOracleSet(tx: Record<string, unknown>): void {
         )
       }
 
-      if (
-        // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access -- we are validating the type
-        'AssetPrice' in priceData.PriceData &&
-        // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access -- we are validating the type
-        !isNumber(priceData.PriceData.AssetPrice)
-      ) {
-        throw new ValidationError('OracleSet: invalid field AssetPrice')
+      const _MINIMUM_ASSET_PRICE_LENGTH = 1
+      const _MAXIMUM_ASSET_PRICE_LENGTH = 16
```

Comment:
Prefixing with `_` is Pythonic. Also move these constant vars outside to the top of the module.
```suggestion
      const MINIMUM_ASSET_PRICE_LENGTH = 1
      const MAXIMUM_ASSET_PRICE_LENGTH = 16
```

Reply:
I was under the belief that private variables are prefixed with the `_` character in Python convention. We use this convention in other constant-variable definitions.

Reply:
I want to limit the scope of these variables, hence I defined them inside this method. I don't see their usage in the other methods of this file. Why should I move it to the top of the file?

Reply:
But we're not using Python, it's TypeScript. I prefer having constants defined at the top of the module for visibility. For example - https://github.com/XRPLF/xrpl.js/blob/c97bbabc9d5bdf0a888b2e82098746f24d6c691a/packages/xrpl/src/models/transactions/oracleSet.ts#L13-L14